ntc_templates/templates/cisco_ios_show_ip_interface_brief.textfsm

```diff
@@ -1,6 +1,6 @@
 Value INTERFACE (\S+)
 Value IP_ADDRESS (\S+)
-Value STATUS (up|down|administratively down)
+Value STATUS (up|down|administratively down|deleted)
```

Comment:
@averhaven
This is more about a sub-interface that is deleted and not a physical interface, correct?